devices/fields/usb/v2_with_LPM.yaml

```diff
@@ -0,0 +1,14 @@
+# Extends USB peripheral with Low-Power Mode
+
+_include:
+  - v2.yaml
+
+LPMCSR:
+  BESL: [0, 0b1111]
+  REMWAKE:
+  LPMACK:
+    Nyet: [0, the valid LPM Token will be NYET]
+    Ack: [1, the valid LPM Token will be ACK]
+  LPMEN:
+    Disabled: [0, enable the LPM support within the USB device]
+    Enabled: [1, no LPM transactions are handled]
```

Comment:
I have not looked through the other RM's but RM0440 under USB_LPMCSR states:

>Bit 0 LPMEN: LPM support enable
>This bit is set by the software to enable the LPM support within the USB device. If this bit is
at ‘**0 no LPM transactions are handled**

Reply:
Thanks.